tortoisebeacon/db_test.go

```diff
@@ -0,0 +1,13 @@
+package tortoisebeacon
+
+import (
+	"testing"
+
+	"github.com/spacemeshos/go-spacemesh/database"
+	"github.com/spacemeshos/go-spacemesh/log/logtest"
+)
+
+func newMemDB(tb testing.TB) *DB {
```

Comment:
is this used?

Reply:
it was used in the test which i didn't submit in this pr. can remove